great_expectations/checkpoint/actions.py

```diff
@@ -269,17 +270,22 @@ def _root_validate_slack_params(cls, values: dict) -> dict:
         return values
 
     @override
-    def _run(  # type: ignore[override] # signature does not match parent  # noqa: C901, PLR0913
+    def _run(  # type: ignore[override] # signature does not match parent  # noqa: C901, PLR0912, PLR0913
         self,
         validation_result_suite: ExpectationSuiteValidationResult,
         validation_result_suite_identifier: Union[ValidationResultIdentifier, GXCloudIdentifier],
-        data_asset=None,
         payload=None,
         expectation_suite_identifier=None,
         checkpoint_identifier=None,
+        checkpoint_result: CheckpointResult | None = None,
     ):
         logger.debug("SlackNotificationAction.run")
 
+        if checkpoint_result:
+            raise NotImplementedError(
+                f"{self.__class__.__name__} does not yet support V1 CheckpointResults."
+            )
```

Comment:
Going to write subsequent PRs to update each of these actions and their respective renderers to work with a checkpoint result